content/fr/docs/setup/independent/ha-topology.md

```diff
@@ -0,0 +1,81 @@
+---
+title: Options pour la topologie en haute disponibilité
+content_template: templates/concept
+weight: 50
+---
+
+{{% capture overview %}}
+
+Cette page explique les deux options de configuration de topologie de vos clusters Kubernetes 
+pour la haute disponibilité.
+
+Vous pouvez configurer un cluster en haute disponibilité:
+
+- Avec des nœuds du control plane empilés, les nœuds etcd étant co-localisés avec des nœuds du control plane
```

Comment:
"co-localisés" ? are you sure of traduction ? "sur les mêmes noeuds" ?

Reply:
yes we use that term here, someone else could give his opinion?

Reply:
co-localisés lgtm